evadb/binder/statement_binder.py

```diff
@@ -69,6 +71,31 @@ def _bind_abstract_expr(self, node: AbstractExpression):
     def _bind_explain_statement(self, node: ExplainStatement):
         self.bind(node.explainable_stmt)
 
+    @bind.register(CreateUDFStatement)
+    def _bind_create_udf_statement(self, node: CreateUDFStatement):
+        if node.query is not None:
+            self.bind(node.query)
+            all_column_list = get_column_definition_from_select_target_list(
+                node.query.target_list
+            )
+            arg_map = {key: value for key, value in node.metadata}
+            assert (
+                "predict" in arg_map
+            ), f"Creating {node.udf_type} UDFs expects 'predict' metadata."
+            # We only support a single predict column for now
+            predict_columns = set([arg_map["predict"]])
+            inputs, outputs = [], []
+            for column in all_column_list:
+                if column.name in predict_columns:
+                    column.name = column.name + "_predictions"
```

Comment:
How do we ensure the UDF always returns something named `xxx_predictions`?

Reply:
The ludwig appends the `_predictions` to the target column. I haven't found a way to specify the output column name for ludwig yet.